### PR TITLE
Release @google-cloud/paginator v0.2.0

### PR DESCRIPTION
**Pull request by callmehiphop**
This pull request was generated using releasetool.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,56 @@
 
 [1]: https://www.npmjs.com/package/nodejs-paginator?activeTab=versions
 
+## v0.2.0
```
Comment (bcoe) [inline CHANGELOG.md]: What's our policy for pre-major releases, i.e., is there a policy for when we'd move to `1.x.x`. I've tended to just immediately release at `1.x.x`, which simplifies the conventional commit workflow a bit (I've always adhered to the philosophy versions are cheap).

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,56 @@
...
+### New Features
+- feat: handle promise based functions ([#91](https://github.com/googleapis/nodejs-paginator/pull/91))
+- refactor(ts): create generic for object streams ([#101](https://github.com/googleapis/nodejs-paginator/pull/101))
+
+### Dependencies
+- chore(deps): update dependency through2 to v3 ([#53](https://github.com/googleapis/nodejs-paginator/pull/53))
+- chore(deps): update dependency @types/is to v0.0.21 ([#55](https://github.com/googleapis/nodejs-paginator/pull/55))
+- chore(deps): update dependency gts to ^0.9.0 ([#57](https://github.com/googleapis/nodejs-paginator/pull/57))
+- fix: Pin @types/sinon to last compatible version ([#61](https://github.com/googleapis/nodejs-paginator/pull/61))
+- refactor: trim a few dependencies ([#60](https://github.com/googleapis/nodejs-paginator/pull/60))
```
Comment (bcoe) [inline CHANGELOG.md]: This format is pretty darn close to what you get out of `standard-version` or `semantic-release`, would be curious to talk about what specific section of the README are necessary. Currently standard-version only displays `feat`, BREAKING CHANGES and `fixes`, we'd need to add the concept of dependency updates, documentation changes, and test changes.

File: CHANGELOG.md (hunk 1 above)
Comment (JustinBeckwith) [reply CHANGELOG.md]: Yup.  This is something we've talked about a lot internally, and we're ready to start doing 1.0 releases *after* node 6 EOL.

Comment (callmehiphop) [reply CHANGELOG.md]: Modules like this are used across a number of our client libraries and serve more as an implementation detail, so I think we can make this 1.x whenever we want. I've always looked to @JustinBeckwith to make that decision.

However for client libraries Googlers tend to be pretty reserved when it comes to making major bumps. Usually 1.x means the library has gone GA and IIRC the request is to try and limit the number of major bumps to a few per year.

File: CHANGELOG.md (hunk 2 above)
Comment (JustinBeckwith) [reply CHANGELOG.md]: Yuuuuup.  The tool that generates this today works across language.  Making the whole conventional-commits / standard-version / semantic-release toolchain work across all languages is a very noble goal for the ecosystem - and something I'd love for our team :) This is a long arc and OKR level discussion though (not for a PR)

Comment (callmehiphop) [reply CHANGELOG.md]: Releases are sort of automated (at least for Node), we use a tool to generate the release notes and publish to npm. Might be worthwhile to open an issue over at https://github.com/googleapis/releasetool

**Comment by callmehiphop** [comment]
@AVaksman just an fyi!

**Pull request merged**